app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -2314,268 +2314,200 @@ not send confidential information to `httpbin.org` that is used here
-            "query",
-            "header",
-            "body"
-        ],
-        "issuer": "<ISSUER>",
-        "leeway": 0,
-        "login_action": "upstream",
-        "login_tokens": [
-            "id_token"
-        ],
-        "response_mode": "query",
-        "reverify": false,
-        "scopes": [
-            "openid"
-        ],
-        "ssl_verify": false,
-        "timeout": 10000,
-        "upstream_access_token_header": "authorization:bearer",
-        "verify_claims": true,
-        "verify_nonce": true,
-        "verify_parameters": true,
-        "verify_signature": true
-    },
-    "created_at": 1502731347000,
+  "config": {
+    "audience_claim": [
+      "aud"
+    ],
+    "client_id": [
+      "kong"
+    ],
+    "client_secret": [
+      "b8068d7d-d7bf-4b23-8724-881ee49bdbfd"
+    ],
+    "consumer_by": [
+      "username",
+      "custom_id"
+    ],
+    "introspect_jwt_tokens": false,
+    "introspection_hint": "access_token",
+    "issuer": "http://ip10-0-0-3-bec2g5kmeb6ge43qmuc0-8080.direct.konglabs-s3.simplru.com/auth/realms/master",
+    "created_at": 1536700540000,
     "enabled": true,
-    "id": "4a91a0ef-1632-491d-a4e3-b8f98f75dcda",
-    "name": "openid-connect"
+    "id": "6836ba3c-12e0-4f2e-bdfd-88134eaa1786",
+    "name": "openid-connect",
+    "service_id": "903978c9-2472-4a04-aff5-d9ba13821e64"
+  }
 }
 ```
 
+#### 4. Verify that Authorization is Now Required
 
-#### 4. Try the API
+Attempt the following command without authorization:
 
 ```bash
-$ http :8000 -v
+$ http get :8000/anything
 ```
 
-As you might have expected, it doesn't work anymore:
-
-```http
-GET / HTTP/1.1
-Accept: */*
-Accept-Encoding: gzip, deflate
-Connection: keep-alive
-Host: localhost:8000
-User-Agent: HTTPie/0.9.9
-```
+It will result in:
 
-as it gives this redirect as a reply:
-
-```http
-HTTP/1.1 302 Moved Temporarily
-Connection: keep-alive
-Content-Length: 167
-Content-Type: text/html
-Date: Mon, 14 Aug 2017 17:24:59 GMT
-Location: https://<ISSUER>/authorize?scope=openid&client_id=<CLIENT_ID>&response_mode=query&state=y2J74-KJFzogFXEtWgwDzl-Y&nonce=J-Ylp3E4dIQIhgutGFo3JOOU&redirect_uri=<REDIRECT_URI>&response_type=code
-Server: kong/0.10.3
-Set-Cookie: authorization=<COOKIE>; Path=/; HttpOnly
 ```
-```html
-<html>
-  <head><title>302 Found</title></head>
-  <body bgcolor="white">
-    <center><h1>302 Found</h1></center>
-  </body>
-</html>
+HTTP 301 Moved Temporarily
 ```
 
 Now, at this point you could try to open the page using a browser and
 see if you can go through the authorization code flow, and after that
-get an reply from httpbin.org. Please check that your redirect uri is
+get an reply from httpbin.org. Please check that your redirect URI is
 correctly registered as the identity provider should redirect the
-browser back to Kong url where this plugin is enabled (it can be the same
-API or it can be different API).
+browser back to Kong URL where this Plugin is enabled (it can be the same
```

Comment:
this sentence is a bit weird, can we clarify it a bit? It's hard to understand for the following portion of the paragraph: "the identity provider should redirect the browser back to Kong URL where the Plugin is enabled". I would write something closer to: 
"Please check that the redirect URI parameter is correctly set as this is where the identity provider will redirect the browser (user?) once the browser/user has authenticated."

Reply:
```suggestion
user back to this URI once authentication is successful. (The URI can be the same
```